tools/truncate-silence.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+tools_path=$(dirname "$0")
+
+usage() {
+    echo "Usage: $0 [OPTION] [FILE]" 1>&2;
+    echo "Truncates silence from the start or the end of an audio file"
+    echo "    -c    Check ffmpeg." 1>&2;
+    echo "    -s    Truncate files from the start of the stream." 1>&2;
+    echo "    -e    Truncate files from the end of the stream." 1>&2;
+    echo "    -r    Amplitud ratio. Samples with lower amplitud than this ratio will be removed." 1>&2;
+    exit 1;
+}
+
+check_ffmpeg=true
+truncate_start=false
+truncate_end=false
+ratio=0.005
+
+while getopts ":cser:" opt; do
```

Comment:
I think `getopt` is preferred over `getopts`, because it handles long options as well.